site/_wiki/Documentation/ConfigurationGuide.md

```diff
@@ -0,0 +1,482 @@
+almost the same indices to probe across many generations.
+
+Here are some recommended starting points for initialization strings, in order
+of most common to least common
+
+- `200`
+- `100`
+
+If `100` works but you have missing auxiliary buttons, you may need to follow
+it up with an string initialization on `123`
+
+The following 2 initialization strings have only been seen once:
+
+- `109`
+- `100` followed by `110`
+
+#### USB Packet capture {#pcap}
+
+This is usually done with Wireshark (and winpcap). A guide on this is planned
+but not in progress. See [opentabletdriver.github.io#171] for more information.
+
+[opentabletdriver.github.io#171]: https://github.com/OpenTabletDriver/opentabletdriver.github.io/issues/171
+
+### Device strings {#device-strings}
+
+In some cases, vendors (especially Huion) might release a tablet with different
+specifications that otherwise has the same ID's as another released tablet.
+
+While this complicates things, we have another tool in our arsenal called
+device strings descriptors.
+
+The configuration format is a dictionary where the key matches the descriptor
+ID (up to 255) and the value is a regex that the string descriptor's value must
+match with.
+
+While they are similar to [initialization strings](#init-strings), the
+difference here is that we read the returned output when initializing the
+string, and as such can match tablets based on its contents.
+
+#### Common device strings {#common-device-strings}
+
+Below are some usually common device strings for various vendors.
+
+These are not guaranteed to be correct across all models for the vendor, but
+may help some people decode their meaning.
+
+| Index | Value |
+| ----: | :---- |
+|   2   | **XP-Pen**: Model Name
+|   4   | **XP-Pen**: Firmware Build ID
+|   5   | **XP-Pen**: Firmware Build String
+|   23  | **VEIKK**: Firmware Build Date
+|  201  | **Gaomon**, **Huion**, and some other **UC-Logic** based: Firmware Build ID
+
+Be wary that probing strings can also change behavior in the device, which can
+usually be rectified by replugging the tablet.
+
+## Validating values {#validation}
+
+Assuming you now have a configuration that correctly identifies your tablet,
+you now have to validate that you're using the correct parser and then the
+correct configuration values.
+
+### Validating the parser {#validation-parser}
+
+This section is **not** for making sure the specifications in the configuration
+is configured correctly, only the parser.
+
+The values below can be validated from entirely within the tablet debugger.
+
+The tablet debugger will show the values as parsed directly from the tablet,
+before any configuration adjustments (such as button count) take place.
+
+If any of the values seems absent or malfunctioning (overflowing, abrupt jumps),
+you are probably using the wrong parser.
+
+#### Digitizer {#validation-parser-digitizer}
+
+Verify that the top left of the tablet equals to `[0, 0]` or `[1, 1]`.
```

Comment:
Probably worth noting that if the user has their tablet rotated physically it may not be the top left.

Reply:
suggest user to rotate tablet, or suggest user to mental math the rotation in their head? This seems not easy to word properly

Reply:
Just a note that it may not be top left if the tablet is rotated. That should be enough to point the user in the right direction.

Reply:
Hopefully resolved in ee90953